docs/developer/Selectors(Test-id) style.md

```diff
@@ -0,0 +1,54 @@
+Selectors style:
+
+1) The file with selectors should have name: 'fileName'.selectors.ts   
+
+Example:  Settings.selectors.ts
+
+
+2) The file with selectors should be in the one directory with its .tsx file
+
+Example: Settings.selectors.ts and Settings.tsx are in the one directory
+
+<img width="200" alt="image" src="https://user-images.githubusercontent.com/95026747/216966156-2749d74f-e8a7-4c2f-913d-573550899f2c.png">
+
+
+
+3) The object with selectors should be "enum" and has a name: 'pageName'Selectors
+
+Example: 
+
+
+<img width="333" alt="image" src="https://user-images.githubusercontent.com/95026747/216958573-a157d971-52bc-4669-8749-e3c6b230962a.png">
+
+
+4) A selector should have construction: selectorName = 'Page name/Page Element'
+
+Example: 
+
+<img width="509" alt="image" src="https://user-images.githubusercontent.com/95026747/216965197-6c6c1b8f-e8ff-4005-bfe8-398af8f902d1.png">
+
+
+
+
+5) "selectorName" - Is a name of selector in the object. should be written in lower case and without "-" , "_" and other symbols
+
+6) "Page name" - Is a name of a page where the element is.
+   Examples:
+     * Home page - "Home/Page Element"
+     * Import Account - "Import Account/Page Element"
+
+7) "/" - Is a separator between page name and page element. No spaces before and after the slash.
+   Examples:
+     * Wrong - "Home /Page Element"
+     * Wrong - "Home/ Page Element"
+     * Wrong - "Home / Page Element"
+     * True - "Home/Page Element"
+
+8) "Page Element" - Is a name of a page element where the element is.
+   Examples:
+     * Import Account Button - "Page name/Import Account Button"
+     * Delegate Tab - "Page name/Delegate Tab"
+     * Private Key Input Field - "Page name/Private Key Input"
```

Comment:
Write the list of available element names

Reply:
element names with pictures are added